Altis_Life.Altis/stringtable.xml

```diff
@@ -5958,7 +5958,7 @@
             <Polish>Ten dom jest już własnością, nawet jeśli nie należy widzieć tę wskazówkę ...</Polish>
             <Portuguese>Esta casa já é de propriedade mesmo que você não deveria estar vendo essa dica ...</Portuguese>
             <Russian>Этот дом уже находится в собственности, даже если вы не должны видеть эту подсказку ...</Russian>
-            <German>Dieses Haus befindet sich bereits im Besitz, auch wenn Sie nicht diesen Hinweis werden sehen ...</German>
+            <German>Dieses Haus befindet sich bereits in deinem Besitz, auch wenn du es nicht sehen solltest...</German>
```

Comment:
Dieses Haus befindet sich bereits in deinem Besitz. **Diesen Hinweis solltest du eigentlich gar nicht sehen...**